extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -188,6 +202,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    this.lifecycleReporter_.sendPing('layoutCallback');
```

Comment:
Though those are functions with same name, but we're doing totally different things in them. I feel it would be an apple to pear comparison.

I would suggest we select some common comparable points between a4a and 3p